tests/testButler.py

```diff
@@ -111,7 +110,7 @@ def getTestDataDir(self):
         try:
             datadir = getPackageDir("testdata_cfht")
         except pexExcept.NotFoundError as e:
-            warnings.warn(e.message)
+            warnings.warn(e.args[0])
```

Comment:
I am guessing that is also why the original code used `e.message` instead of `str(e)`.